File: Programmers/Python/정렬_K번째수.py
def solution(array, commands):
    answer = []
    
    for command in commands:
        sorted_array = sorted(array[command[0]-1:command[1]])
        # array[:command[0]-1] + sorted(array[command[0]-1:command[1]]) + array[command[1]:]
        answer.append(sorted_array[command[2]-1])
        
    return answer

File: Programmers/JavaScript/정렬_K번째 수.js
function solution(array, commands) {
    var answer = [];
    var temp = [];
    for (var i=0 ; i < commands.length ; i++){
        temp = array.slice(commands[i][0]-1, commands[i][1]);
        temp.sort(function(a,b){
            return a-b;
        });
        // console.log(temp);
        answer.push(temp[commands[i][2]-1]);
    }
    return answer;
}

// arr.sort([compareFunction])
// compareFunction 값이 생략되면, 배열의 element들은 문자열로 취급되어, 유니코드 값 순서대로 정렬된다.
// 두 개의 배열 elememt(ex. a,b)를 파라미터로 입력 받아, 
// return 값이 0보다 작으면 a,b 의 순으로 / 0보다 크면 b,a / 같으면 순서를 변경하지 않는다.